### PR TITLE
Kp/remove solution

### PR DESCRIPTION
**Pull request by kyra-patton**
Removed solution and added a Readme describing the problem.

### CODE REVIEW

**Comment by kyra-patton** [comment]
Removes [solution](https://github.com/AdaGold/dijkstra/tree/solution) and updates readme

File: README.md [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
+    {
+        "start_node": 0, 
+        "parent_list": [None, 0, 1, 2, 5, 6, 7, 0, 2], 
+        "shortest_distances": [0, 4, 12, 19, 21, 11, 9, 8, 14]
+    }
+```
+
+## Note
+
+More tests beyond what is provided are needed to verify correctness.
```
Comment (beccaelenzil) [inline README.md]:
```suggestion
The unit tests provided don't cover all test cases needed to verify correctness.
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
-This repository is a sample solution of Dijsktra's algorithm written in Ruby.
+ Dijkstra's algorithm for finding the shortest path from a start node to all other nodes in a graph.
 
-## Todo Items
+## Description
 
-- More tests are needed to verify correctness
+Given an adjacency matrix representation of the graph and the node to start the algorithm from, find the shortest path from a start node to all other nodes in the graph. 
+
+### Example 1
```
Comment (beccaelenzil) [inline README.md:9]: Consider replacing with smaller matrix from test cases

**Review by beccaelenzil — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,7 +1,36 @@
...
+    start_node = 0
+
+Output:
+    {
+        "start_node": 0, 
+        "parent_list": [None, 0, 1, 2, 5, 6, 7, 0, 2], 
+        "shortest_distances": [0, 4, 12, 19, 21, 11, 9, 8, 14]
+    }
+```
+
```
Comment (kyra-patton) [inline README.md]:
```suggestion
Input:
    adjacency_matrix =[ 
      [0, 4, 0, 0],
      [4, 0, 12, 0],
      [0, 12, 0, 0],
      [0,  0, 0, 0]
    ]
Output:
    {
        "start_node": 3, 
        "parent_list": [None, None, None, None], 
        "shortest_distances": [float('inf'), float('inf'), float('inf'), 0]
    }

```

**Pull request merged**